src/components/Card/Card.js

```diff
@@ -2,35 +2,53 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import './Card.css';
 
-const Card = ({ location, districtData, select, id, selected}) => {
-  const years = Object.keys(districtData).map((year, index) => {
-    let style = districtData[year] >= 0.5 ? 'high' : 'low';
-    return (
-      <li className={style}  key={`year-${index}`}>
-        {' '}
-        {year}: {districtData[year]}
-      </li>
-    );
-  });
+const Card = (props) => {
+  let renderedCard;
+  const {isSelected, onSelect} = props;
+  const klass = isSelected ? 'card selected' : 'card';
 
-  const handleCardClick = () => {
-    select(location);
-  };
+  if (!props.compared) {
```

Comment:
Wow, great job!

Reply:
We worked out this logic together! Good work in thinking of how we can do it!